src/scenes/Home/OnboardingHomeView/OnboardingCard.tsx

```diff
@@ -0,0 +1,563 @@
+                    <Typography
+                      component='p'
+                      variant='h6'
+                      sx={{
+                        color: theme.palette.TwClrTxt,
+                        fontSize: '16px',
+                        fontWeight: 600,
+                        lineHeight: '24px',
+                        textAlign: 'center',
+                      }}
+                    >
+                      {strings.COMPLETE}
+                    </Typography>
+                  )}
+                </Box>
+              </Box>
+            ) : (
+              <Box
+                display={'flex'}
+                flexDirection={'column'}
+                sx={{
+                  background: theme.palette.TwClrBgSecondary,
+                  borderRadius: '8px',
+                  opacity: row.enabled ? 1 : 0.5,
+                  marginTop: 2,
+                }}
+              >
+                <Box
+                  key={index}
+                  sx={{
+                    marginTop: '16px',
+                    marginBottom: '16px',
+                    marginLeft: '0px',
+                    marginRight: '0px',
+                    display: 'flex',
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                    width: '100%',
+                    borderRadius: '8px',
+                    paddingTop: '8px',
+                    paddingBottom: '8px',
+                    position: 'relative',
+                  }}
+                >
+                  <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'center', alignItems: 'center' }}>
+                    {isAdmin(selectedOrganization) && (
+                      <Box
+                        sx={{
+                          width: '46px',
+                          height: '46px',
+                          borderRadius: '50%',
+                          backgroundColor: theme.palette.TwClrBgBrand,
+                          color: 'white',
+                          display: 'flex',
+                          justifyContent: 'center',
+                          alignItems: 'center',
+                          fontSize: '20px',
+                          fontWeight: '700',
+                          marginLeft: '16px',
+                        }}
+                      >
+                        {index + 1}
+                      </Box>
+                    )}
+                    <Box
+                      sx={{
+                        background: theme.palette.TwClrBaseGray025,
+                        borderRadius: '8px',
+                        display: 'flex',
+                        height: '88px',
+                        width: '88px',
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                        padding: '8px',
+                        marginLeft: '24px',
+                        marginRight: '24px',
+                      }}
```

Comment:
This page has many inline styles with some overlap between components. Can the shared styles be consolidated?